<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>239</width>
      <height>48</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="NextButton">
        <property name="text">
         <string>Next &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="FilterButton">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="FilteredList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>571</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LikeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Like</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>163</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="ShowCSVButton">
       <property name="text">
        <string>Show CSV file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ShowHTMLButton">
       <property name="text">
        <string>Show HTML file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="MovieListLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Movie list:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilteredLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filtered list:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilterByTitleLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FilterLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter by genre (&quot;empty&quot; to get all):</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MovieList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UndoButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RedoButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
